build-system/tasks/release/index.js

```diff
@@ -456,12 +511,13 @@ async function cleanup_(tempDir) {
 async function release() {
   const outputDir = path.resolve(argv.output_dir || './release');
   const tempDir = path.join(outputDir, 'tmp');
+  const useCustomConfigs = Boolean(argv.use_custom_configs);
```

Comment:
1. The rest of this file uses `argv.flag_name` inside the various functions directly, instead of passing it between the function calls. Let's keep that consistent.
2. We tend to prefer `!!var` instead of `Boolean(var)`

Reply:
Item 1 addressed in commit fccb400.
Item 2 moot once item 1 was addressed.